instrumentation/spring/spring-boot-autoconfigure/build.gradle.kts

```diff
@@ -22,7 +21,7 @@ sourceSets {
 }
 
 dependencies {
-  implementation("org.springframework.boot:spring-boot-autoconfigure:$springBootVersion")
+  compileOnly("org.springframework.boot:spring-boot-autoconfigure:$springBootVersion")
```

Comment:
why is this needed?

Reply:
the spring version was used in the tests - but the user should include the spring starter directly